types/index.test-d.ts

```diff
@@ -23,59 +30,69 @@ const transloadit = new Transloadit({
-  onAssemblyProgress: (assembly) => {
-    expectType<Assembly>(assembly)
-  },
-  onUploadProgress: ({ uploadedBytes, totalBytes }) => {
-    expectType<number | undefined>(uploadedBytes)
-    expectType<number | undefined>(totalBytes)
-  },
-  assemblyId: '123',
-}))
+expectType<Promise<Assembly>>(
+  transloadit.createAssembly({
+    params: {
+      steps: { foo: 'bar' },
+      template_id: 'template',
+      notify_url: 'url',
+      fields: { a: 'b', c: 1 },
+      allow_steps_override: false,
+    },
+    files: {
+      file1: '/path/to/file',
+    },
+    uploads: {
+      file2: 'string',
+      file3: intoStream('string'),
+      file4: Buffer.from('string'),
+    },
+    isResumable: true,
+    chunkSize: Infinity,
+    uploadConcurrency: 5,
+    timeout: 1,
+    waitForCompletion: true,
+    onAssemblyProgress: (assembly) => {
+      expectType<Assembly>(assembly)
+    },
+    onUploadProgress: ({ uploadedBytes, totalBytes }) => {
+      expectType<number | undefined>(uploadedBytes)
+      expectType<number | undefined>(totalBytes)
+    },
+    assemblyId: '123',
+  })
+)
 
-expectType<Promise<Assembly>>(transloadit.awaitAssemblyCompletion('1', {
-  onAssemblyProgress: (assembly) => {
-    expectType<Assembly>(assembly)
-  },
-  timeout: 1,
-  interval: 1,
-}))
+expectType<Promise<Assembly>>(
+  transloadit.awaitAssemblyCompletion('1', {
+    onAssemblyProgress: (assembly) => {
+      expectType<Assembly>(assembly)
+    },
+    timeout: 1,
+    interval: 1,
+  })
+)
 
 expectType<Promise<Assembly>>(transloadit.cancelAssembly('1'))
-expectType<Promise<ReplayedAssembly>>(transloadit.replayAssembly('1', { param1: { a: 1 }}))
-expectType<Promise<{ ok: string, success: boolean }>>(transloadit.replayAssemblyNotification('1', { param1: { a: 1 }}))
-expectType<Promise<{ count: number, items: ListedAssembly[] }>>(transloadit.listAssemblies({ param1: { a: 1 }}))
-expectType<Readable>(transloadit.streamAssemblies({ param1: { a: 1 }}))
+expectType<Promise<ReplayedAssembly>>(transloadit.replayAssembly('1', { param1: { a: 1 } }))
+expectType<Promise<{ ok: string; success: boolean }>>(
+  transloadit.replayAssemblyNotification('1', { param1: { a: 1 } })
+)
+expectType<Promise<{ count: number; items: ListedAssembly[] }>>(
+  transloadit.listAssemblies({ param1: { a: 1 } })
+)
+expectType<Readable>(transloadit.streamAssemblies({ param1: { a: 1 } }))
 expectType<Promise<Assembly>>(transloadit.getAssembly('1'))
 
-expectType<Promise<TemplateResponse>>(transloadit.createTemplate({ param1: { a: 1 }}))
-expectType<Promise<TemplateResponse>>(transloadit.editTemplate('1', { param1: { a: 1 }}))
-expectType<Promise<{ ok: string, message: string }>>(transloadit.deleteTemplate('1'))
+expectType<Promise<TemplateResponse>>(transloadit.createTemplate({ param1: { a: 1 } }))
+expectType<Promise<TemplateResponse>>(transloadit.editTemplate('1', { param1: { a: 1 } }))
+expectType<Promise<{ ok: string; message: string }>>(transloadit.deleteTemplate('1'))
```

Comment:
are comma and semicolon interchangeable in typescript?

Reply:
no expert, in this case they are :)